src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -136,7 +136,7 @@ struct CPUStreamsExecutor::Impl {
                     ? _impl->_config._streams_info_table[_impl->_config._stream_ids[stream_id]][THREADS_PER_STREAM]
                     : 0;
             const auto cpu_core_type =
-                _impl->_config._streams_info_table.size() > 0
+                (_impl->_config._streams_info_table.size() > 0 && _impl->_config._stream_ids.size() > 0)
```

Comment:
why stream IDs are empty?

Reply:
It is just a protection, It should not be empty in normal. The root cause of CoreThreadingTestsWithIterations failure is in PR17240.